plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergModule.java

```diff
@@ -48,7 +47,6 @@ public void configure(Binder binder)
     {
         binder.bind(IcebergTransactionManager.class).in(Scopes.SINGLETON);
         configBinder(binder).bindConfig(IcebergConfig.class);
-        configBinder(binder).bindConfig(HiveMetastoreConfig.class);
```

Comment:
I think you should be able to just do the same in Delta, right?
you did all the Delta changes anyway.

Just update the commit message and PR title when doing so.